packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -416,13 +416,25 @@ export class OrderedClientElection
 			change = true;
 		}
 		if (change) {
+			this.logger.sendTelemetryEvent({
+				eventName: "SummarizerClientElected",
+				electedClientId: this._electedClient?.clientId,
+				electedParentId: this._electedParent?.clientId,
+				sequenceNumber,
```

Comment:
name this `electionSequenceNumber` so that it doesn't get confused with `sequenceNumber` in logs (which will get overwritten by this).

Reply:
yes, I was hoping to use an existing column as not to add a new column just for this

Reply:
Just saw we have electionSequenceNumber  already!